prowler/providers/microsoft365/services/entra/entra_service.py

```diff
@@ -13,14 +13,16 @@ def __init__(self, provider: Microsoft365Provider):
         super().__init__(provider)
 
         loop = get_event_loop()
-
+        self.tenant_domain = provider.identity.tenant_domain
         attributes = loop.run_until_complete(
             gather(
                 self._get_authorization_policy(),
+                self._get_admin_consent_poolicy(),
```

Comment:
```suggestion
                self._get_admin_consent_policy(),
```
prob will need some _search&replace_, as it spread across the file